tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -82,6 +82,11 @@ var _ = Describe("Configuration testing for DPA Custom Resource", func() {
 						Key: bslCredential.Key,
 					}
 				}
+			} else {
```

Comment:
this is in wrong place

it should be in here https://github.com/stillalearner/oadp-operator/blob/fix_3050_cp1.3/tests/e2e/dpa_deployment_suite_test.go#L120

This happens because DPA is going to be reconciled only here https://github.com/stillalearner/oadp-operator/blob/fix_3050_cp1.3/tests/e2e/dpa_deployment_suite_test.go#L99

Reply:
ah silly

Reply:
Thanks @mateusoliveira43